src/Workspaces/Core/Portable/Rename/ConflictEngine/DeclarationConflictHelpers.cs

```diff
@@ -18,7 +18,7 @@ public static ImmutableArray<Location> GetMembersWithConflictingSignatures(IMeth
                                             .OfType<IMethodSymbol>()
                                             .Where(m => !m.Equals(renamedMethod) && m.Arity == renamedMethod.Arity);
 
-            return GetConflictLocations(renamedMethod, potentiallyConfictingMethods, isMethod:true,
+            return GetConflictLocations(renamedMethod, potentiallyConfictingMethods, isMethod: true,
```

Comment:
:grey_exclamation: This file should not be modified in this commit. It is a formatting-only change mixed in with a refactoring that does not need to touch this file.

:memo: The IDE0055 warning is preferable to mixing formatting changes into unrelated refactorings.

Reply:
These are errors locally when i compile.  So i fixed so i wouldn't be blocked.  I can remove from the PR though.